src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitResultContext.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.shortcircuit.ShortCircuitParameters;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.support.MessageBuilder;
+
+import java.io.UncheckedIOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+public class ShortCircuitResultContext {
+
+    private static final String REPORT_UUID = "reportUuid";
+
+    public static final String VARIANT_ID = "variantId";
+
+    private final UUID resultUuid;
+
+    private final ShortCircuitRunContext runContext;
+
+    public ShortCircuitResultContext(UUID resultUuid, ShortCircuitRunContext runContext) {
+        this.resultUuid = Objects.requireNonNull(resultUuid);
+        this.runContext = Objects.requireNonNull(runContext);
+    }
+
+    public UUID getResultUuid() {
```

Comment:
Why don't you use lombok @Getter on these properties ?

Reply:
done